src/trace-plugin-interface.js

```diff
@@ -209,10 +223,32 @@ PluginAPI.prototype.runInRootSpan = function(options, fn) {
 
 /**
  * Convenience method which obtains a Transaction object with getTransaction()
+ * and creates a new child span nested within this transaction. If there is
+ * no current Transaction object, this function returns null.
+ * @param {object} options An object that specifies options for how the child
+ * span is created and propogated. @see Transaction.prototype.createChildSpan
+ * @returns A new ChildSpan object, or null if there is no active root span.
+ */
+PluginAPI.prototype.createChildSpan = function(options, fn) {
```

Comment:
Ideally it should reaplce runInChildSpan. I don't see any reason to keep it around otherwise. Except that it's "semver major" in a sense. Though no one is using it.